### PR TITLE
[ENDPOINT] update news PUT /news/:id OT245 - 48

### PR DESCRIPTION
**Pull request by marattagian**
[OT245 - 48](https://alkemy-labs.atlassian.net/browse/OT245-48)

**AS** admin user
I **WANT TO** update an existent new
**SO** I can keep information updated

**Acceptance Criteria:**
PUT `/news/:id` - Should validate the existence of the new with the given id in the parameters and then update the new and return it with the updated values. Otherwise should return an error.

#### Success Case
Screenshot from 2022-07-23 15-05-17

Screenshot from 2022-07-23 15-06-11

Screenshot from 2022-07-23 15-05-43

Screenshot from 2022-07-23 15-06-50

#### Error Case
Screenshot from 2022-07-23 15-10-26

### CODE REVIEW

**Review by gabodmc — APPROVED** [review]
Bien resuelto!

**Review by R-Berrocal — APPROVED** [review]

File: services/news.js [hunk 1]
```diff
@@ -1,11 +1,39 @@
...
     const createNew = await New.create(body)
     return createNew
   } catch (error) {
     throw new ErrorObject(error.message, error.statusCode || 500)
   }
 }
+
+exports.getNewById = async (id) => {
+  try {
+    const user = await New.findByPk(id)
```
Comment (R-Berrocal) [inline services/news.js]: Acá es new y retornas new, cierto?

**Review by R-Berrocal — CHANGES_REQUESTED** [review]

File: services/news.js (hunk 1 above)
Comment (marattagian) [reply services/news.js]: Una confusión en el naming

**Review by R-Berrocal — APPROVED** [review]

File: services/news.js (hunk 1 above)
Comment (R-Berrocal) [reply services/news.js]: Suele pasar jajajja

**Review by HinojosaAcevedoJ — APPROVED** [review]

**Pull request merged**